Since version 0.9.26 got error "Module not found"

<!-- PLEASE READ THE FOLLOWING INSTRUCTIONS -->

<!--
- Try to search for your issue, it may have already been answered or even fixed in the development version.
- Check if the issue is reproducible with the latest stable version of Buefy.
- It is **required** that you clearly describe the steps necessary to reproduce the issue.
- It is recommended that you make a screenshots/animated GIFs/JSFiddle/JSBin/Codepen to demonstrate your issue. If possible, please use this [CodePen template](https://codepen.io/service-paradis/pen/KKgJZOK) to reproduce the problem.
- If your issue is resolved but still open, don’t hesitate to close it. In case you found a solution by yourself, it could be helpful to explain how you fixed it.
- Use English for communication
-->

### Overview of the problem

**Buefy** version: [0.9.26]
**Vuejs** version: [2.7.16]
**OS/Browser**:

### Description

Updating from 0.9.23 to 0.9.26 gave me the error "Module not found".
0.9.25 is the last version that seems to work well.

### Steps to reproduce

1. Update to 0.9.26
2. ``npm i```
3. Build application

<!--
1. First Step
4. Second Step
5. and so on...
-->

### Expected behavior

<!--What you expected to happen-->

### Actual behavior

```ERROR in ./src/main.js 2:0-26
Module not found: Error: Can't resolve 'buefy' in '/Users/julianitow/medissimo/arbok/src'
resolve 'buefy' in '/Users/julianitow/medissimo/arbok/src'
  Parsed request is a module
  using description file: /Users/julianitow/medissimo/arbok/package.json (relative path: ./src)
    Field 'browser' doesn't contain a valid alias configuration
    resolve as module
      /Users/julianitow/medissimo/arbok/src/node_modules doesn't exist or is not a directory
      looking for modules in /Users/julianitow/medissimo/arbok/node_modules
        single file module
          using description file: /Users/julianitow/medissimo/arbok/package.json (relative path: ./node_modules/buefy)
            no extension
              Field 'browser' doesn't contain a valid alias configuration
              /Users/julianitow/medissimo/arbok/node_modules/buefy is not a file
            .js
              Field 'browser' doesn't contain a valid alias configuration
              /Users/julianitow/medissimo/arbok/node_modules/buefy.js doesn't exist
            .vue
              Field 'browser' doesn't contain a valid alias configuration
              /Users/julianitow/medissimo/arbok/node_modules/buefy.vue doesn't exist
            .json
              Field 'browser' doesn't contain a valid alias configuration
              /Users/julianitow/medissimo/arbok/node_modules/buefy.json doesn't exist
        existing directory /Users/julianitow/medissimo/arbok/node_modules/buefy
          using description file: /Users/julianitow/medissimo/arbok/node_modules/buefy/package.json (relative path: .)
            using description file: /Users/julianitow/medissimo/arbok/package.json (relative path: ./node_modules/buefy)
              no extension
                Field 'browser' doesn't contain a valid alias configuration
                /Users/julianitow/medissimo/arbok/node_modules/buefy is not a file
              .js
                Field 'browser' doesn't contain a valid alias configuration
                /Users/julianitow/medissimo/arbok/node_modules/buefy.js doesn't exist
              .vue
                Field 'browser' doesn't contain a valid alias configuration
                /Users/julianitow/medissimo/arbok/node_modules/buefy.vue doesn't exist
              .json
                Field 'browser' doesn't contain a valid alias configuration
                /Users/julianitow/medissimo/arbok/node_modules/buefy.json doesn't exist
              as directory
                existing directory /Users/julianitow/medissimo/arbok/node_modules/buefy
                  using description file: /Users/julianitow/medissimo/arbok/node_modules/buefy/package.json (relative path: .)
                    use ./dist/esm/index.js from module in package.json
                      using description file: /Users/julianitow/medissimo/arbok/node_modules/buefy/package.json (relative path: ./dist/esm/index.js)
                        no extension
                          Field 'browser' doesn't contain a valid alias configuration
                          /Users/julianitow/medissimo/arbok/node_modules/buefy/dist/esm/index.js doesn't exist
                        .js
                          Field 'browser' doesn't contain a valid alias configuration
                          /Users/julianitow/medissimo/arbok/node_modules/buefy/dist/esm/index.js.js doesn't exist
                        .vue
                          Field 'browser' doesn't contain a valid alias configuration
                          /Users/julianitow/medissimo/arbok/node_modules/buefy/dist/esm/index.js.vue doesn't exist
                        .json
                          Field 'browser' doesn't contain a valid alias configuration
                          /Users/julianitow/medissimo/arbok/node_modules/buefy/dist/esm/index.js.json doesn't exist
                        as directory
                          /Users/julianitow/medissimo/arbok/node_modules/buefy/dist/esm/index.js doesn't exist
                    use ./dist/cjs/index.js from main in package.json
                      using description file: /Users/julianitow/medissimo/arbok/node_modules/buefy/package.json (relative path: ./dist/cjs/index.js)
                        no extension
                          Field 'browser' doesn't contain a valid alias configuration
                          /Users/julianitow/medissimo/arbok/node_modules/buefy/dist/cjs/index.js doesn't exist
                        .js
                          Field 'browser' doesn't contain a valid alias configuration
                          /Users/julianitow/medissimo/arbok/node_modules/buefy/dist/cjs/index.js.js doesn't exist
                        .vue
                          Field 'browser' doesn't contain a valid alias configuration
                          /Users/julianitow/medissimo/arbok/node_modules/buefy/dist/cjs/index.js.vue doesn't exist
                        .json
                          Field 'browser' doesn't contain a valid alias configuration
                          /Users/julianitow/medissimo/arbok/node_modules/buefy/dist/cjs/index.js.json doesn't exist
                        as directory
                          /Users/julianitow/medissimo/arbok/node_modules/buefy/dist/cjs/index.js doesn't exist
                    using path: /Users/julianitow/medissimo/arbok/node_modules/buefy/index
                      using description file: /Users/julianitow/medissimo/arbok/node_modules/buefy/package.json (relative path: ./index)
                        no extension
                          Field 'browser' doesn't contain a valid alias configuration
                          /Users/julianitow/medissimo/arbok/node_modules/buefy/index doesn't exist
                        .js
                          Field 'browser' doesn't contain a valid alias configuration
                          /Users/julianitow/medissimo/arbok/node_modules/buefy/index.js doesn't exist
                        .vue
                          Field 'browser' doesn't contain a valid alias configuration
                          /Users/julianitow/medissimo/arbok/node_modules/buefy/index.vue doesn't exist
                        .json
                          Field 'browser' doesn't contain a valid alias configuration
                          /Users/julianitow/medissimo/arbok/node_modules/buefy/index.json doesn't exist
      /Users/julianitow/medissimo/node_modules doesn't exist or is not a directory
      /Users/julianitow/node_modules doesn't exist or is not a directory
      /Users/node_modules doesn't exist or is not a directory
      /node_modules doesn't exist or is not a directory
```
<!--What actually happened-->
<!--If possible, please use this [CodePen template](https://codepen.io/service-paradis/pen/KKgJZOK) to reproduce the problem.-->

--- a/.github/workflows/npm_deploy.yml
+++ b/.github/workflows/npm_deploy.yml
@@ -10,14 +10,15 @@ jobs:
       contents: read
       packages: write
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       # Setup .npmrc file to publish to GitHub Packages
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
         with:
-          node-version: '16.x'
+          node-version: '18.x'
           registry-url: 'https://registry.npmjs.org/'
           scope: 'buefy'
       - run: npm ci
+      - run: npm run build:lib
       - run: npm publish --access public
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "buefy",
-  "version": "0.9.26",
+  "version": "0.9.27",
   "homepage": "https://buefy.org",
   "description": "Lightweight UI components for Vue.js based on Bulma",
   "author": "Rafael Beraldo <rafael.pimpa@gmail.com>",

